packages/lib/hooks/useTraceUpdate.tsx

```diff
@@ -0,0 +1,21 @@
+import { useRef, useEffect } from "react";
+
+export function useTraceUpdate(props: { [s: string]: unknown } | ArrayLike<unknown>) {
```

Comment:
Helper to be used while debugging - call it in any component and see why it it updated/ how many times it re-rendered